### PR TITLE
Password Lock

### PR DESCRIPTION
**Pull request by Ada18980**
Adds a new lock, similar to the combination lock. However, instead of being limited to only numbers, it is limited to only 6 letter passwords made of letters, and the user can set a hint. Once a password is set, it cannot be changed.

Additionally, futuristic items will use the password entered by players and can be unlocked if the password entered in the field is the same.

Also changes the main hall maid to remove the lock from chastity belts.

### CODE REVIEW

File: BondageClub/Scripts/Server.js [hunk 1]
```diff
@@ -391,6 +409,9 @@ function ServerAppearanceLoadFromBundle(C, AssetFamily, Bundle, SourceMemberNumb
 							if (C.Appearance[A].Property.LockedBy != null) NA.Property.LockedBy = C.Appearance[A].Property.LockedBy;
 							if (C.Appearance[A].Property.LockMemberNumber != null) NA.Property.LockMemberNumber = C.Appearance[A].Property.LockMemberNumber; else delete NA.Property.LockMemberNumber;
 							if (C.Appearance[A].Property.CombinationNumber != null) NA.Property.CombinationNumber = C.Appearance[A].Property.CombinationNumber; else delete NA.Property.CombinationNumber;
+							if (C.Appearance[A].Property.Password != null) NA.Property.CombinationNumber = C.Appearance[A].Property.Password; else delete NA.Property.Password;
```
Comment (Nina-1474) [inline BondageClub/Scripts/Server.js:412]: I'm guessing this shouldn't be `NA.Property.CombinationNumber` every time.

**Review by Nina-1474 — DISMISSED** [review]

File: BondageClub/Scripts/Server.js (hunk 1 above)
Comment (Ada18980) [reply BondageClub/Scripts/Server.js:412]: Whoops

Comment (Nina-1474) [reply BondageClub/Scripts/Server.js]: The line I highlighted was just one of them. There's another few farther down.

Comment (Ada18980) [reply BondageClub/Scripts/Server.js]: Fixed! Thanks >.<

**Comment by Ben987** [comment]
Alright!  Time to test this one, thanks a lot for that new lock :)

**Pull request merged**